src/targets/gpu/include/migraphx/gpu/hip.hpp

```diff
@@ -81,6 +83,30 @@ struct hip_allocate
     }
 };
 
+struct hip_fill
+{
+    int value = 0;
+
+    template <class Self, class F>
+    static auto reflect(Self& self, F f)
+    {
+        return pack(f(self.value, "value"));
+    }
+
+    std::string name() const { return "hip::fill"; }
+    shape compute_shape(const std::vector<shape>& inputs) const
+    {
+        check_shapes{inputs, *this}.has(1);
```

Comment:
assert for packed tensor. As hipMemsetAsync can't handle non-packed

Reply:
Can you check for it ?

Reply:
I added an `assert` in the `hip_fill` function.